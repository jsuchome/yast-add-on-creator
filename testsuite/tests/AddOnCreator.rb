# encoding: utf-8

module Yast
  class AddOnCreatorClient < Client
    def main
      # testedfiles: AddOnCreator.ycp

      Yast.include self, "testsuite.rb"
      TESTSUITE_INIT([], nil)

      Yast.import "AddOnCreator"

      DUMP("AddOnCreator::Modified")
      TEST(lambda { AddOnCreator.Modified }, [], nil)

      nil
    end
  end
end

Yast::AddOnCreatorClient.new.main
