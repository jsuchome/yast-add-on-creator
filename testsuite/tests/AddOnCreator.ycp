{

// testedfiles: AddOnCreator.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "AddOnCreator";

DUMP("AddOnCreator::Modified");
TEST(``(AddOnCreator::Modified()), [], nil);

}
