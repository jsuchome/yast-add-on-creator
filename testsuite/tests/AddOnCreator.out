Dump	AddOnCreator::Modified
Return	false
